### DISCUSSION TITLE
Upgraded to Pro account. Application still encountering 10 second timeout

### DISCUSSION THREAD

**Discussion by delaneyp-irl** [vercel/community#725] [Help] [answered]
I have created an application that makes an external web service call which takes longer than 10 seconds to respond. I upgraded to a Pro account as I thought this account has a 60 second timeout limit. When the application using the new Pro account make the call to the same external web service Vercel responds with a timeout response after 10 seconds.

Is there a configuration change I must make in order to get the 60 second timeout for a request/response?

**Comment by JTaylor0196** [comment, marked as answer]
Hey @delaneyp-irl!

Can you ensure that you've:
1) Transferred the project from your Hobby account to your new Pro Team?
2) Redeployed the project from your dashboard after transferring it to the Pro Team?
